scripts/build.sh

```diff
@@ -2,6 +2,20 @@
 
 set -e
 
+for arg in "$@"
+do
+    case $arg in
+        --quick)
+            QUICK_BUILD=1
```

Comment:
Should make this emit a disclaimer saying that if it doesn't work run a proper build